lib/reboot_cmds.py

```diff
@@ -77,26 +78,30 @@ def refresh_contract(cfg: config.UAConfig):
 
 
 def main(cfg: config.UAConfig) -> int:
```

Comment:
what if separate main into two blocks:

One that executes the previous behavior tied to the marker file
One that executes the `onlySeries` part

In that sense, we make that separation clearer and we simplify the code a bit. What do you think ?

Reply:
The only reason I was a little skeptical about breaking it into two was the `lock` which is already being held by `pro-reboot-cmds`.

I am unsure of having two locks for a similar flow of events since the onlySeries bit is just a function call?  :sweat_smile: 
Let me know what you think!

Reply:
Updated @lucasmoura @renanrodrigo